content/pt/docs/reference/kubectl/cheatsheet.md

```diff
@@ -0,0 +1,387 @@
+`.yml`, e `.json` pode ser usado.
+
+```bash
+kubectl apply -f ./my-manifest.yaml            # criar recurso (s)
+kubectl apply -f ./my1.yaml -f ./my2.yaml      # criar a partir de vários arquivos
+kubectl apply -f ./dir                         # criar recurso (s) em todos os arquivos de manifesto no diretório
+kubectl apply -f https://git.io/vPieo          # criar recurso (s) a partir de URL
+kubectl create deployment nginx --image=nginx  # iniciar uma única instância do nginx
+kubectl explain pods,svc                       # obtenha a documentação para pod e svc
+
+# Crie vários objetos YAML a partir de stdin
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: busybox-sleep
+spec:
+  containers:
+  - name: busybox
+    image: busybox
+    args:
+    - sleep
+    - "1000000"
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: busybox-sleep-less
+spec:
+  containers:
+  - name: busybox
+    image: busybox
+    args:
+    - sleep
+    - "1000"
+EOF
+
+# Crie um segredo com várias chaves
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysecret
+type: Opaque
+data:
+  password: $(echo -n "s33msi4" | base64 -w0)
+  username: $(echo -n "jane" | base64 -w0)
+EOF
+
+```
+
+## Visualizando e localizando recursos
+
+```bash
+# Obter comandos com saída simples
+kubectl get services                          # Listar todos os serviços do namespace
+kubectl get pods --all-namespaces             # Listar todos os pods em todos namespaces
+kubectl get pods -o wide                      # Listar todos os pods no atual namespace, com mais detalhes
+kubectl get deployment my-dep                 # Listar uma implantação específica
+kubectl get pods                              # Listar todos os pods no namespace
+kubectl get pod my-pod -o yaml                # Obter o YAML de um pod
+kubectl get pod my-pod -o yaml --export       # Obtenha o YAML de um pod sem informações específicas do cluster
+
+# Descrever comandos com saída detalhada
+kubectl describe nodes my-node
+kubectl describe pods my-pod
+
+# Listar serviços classificados por nome
+kubectl get services --sort-by=.metadata.name
+
+# Listar pods classificados por contagem de reinicializações
+kubectl get pods --sort-by='.status.containerStatuses[0].restartCount'
+
+# Lista PersistentVolumes classificado por capacidade
+kubectl get pv --sort-by=.spec.capacity.storage
+
+# Obtenha a versão da label de todos os pods com a label app=cassandra
+kubectl get pods --selector=app=cassandra -o \
+  jsonpath='{.items[*].metadata.labels.version}'
+
+# Obter todos os nós dos workers  (use um seletor para excluir resultados que possuem um rótulo
+# nomeado 'node-role.kubernetes.io/master')
+kubectl get node --selector='!node-role.kubernetes.io/master'
+
+# Obter todos os pods em execução no espaço para nome
+kubectl get pods --field-selector=status.phase=Running
+
+# Obter ExternalIPs de todos os nós
```

Comment:
```suggestion
# Obter IPs externos de todos os nós
```

Reply:
ExternalIPs is the type, so we should not translate.